source/auth/tests/README.rst

```diff
@@ -0,0 +1,52 @@
+==========
+Auth Tests
+==========
+
+The YAML and JSON files in this directory tree are platform-independent tests
+that drivers can use to prove their conformance to the Auth Spec at least with
+respect to connection string URI input.
+
+Drivers should do additional unit testing if there are alternate ways of
+configuring credentials on a client.
+
+Driver must also conduct the prose tests in the Auth Spec test plan section.
+
+Format
+------
+
+Each YAML file contains an object with a single ``tests`` key. This key is an
+array of test case objects, each of which have the following keys:
+
+- ``description``: A string describing the test.
+- ``uri``: A string containing the URI to be parsed.
+- ``valid:`` A boolean indicating if the URI should be considered valid.
+- ``credential_configured:`` A boolean indicating if the URI is considered to have
```

Comment:
I suppose this addresses my original question in https://github.com/mongodb/specifications/pull/532/files#r273202088.